classes/class-swsales-metaboxes.php

```diff
@@ -313,6 +313,26 @@ public static function display_step_type( $post ) {
 				// Filter to add custom settings from module .
 				do_action( 'swsales_after_choose_sale_type', $cur_sale );
 				?>
+				<tr>
+					<th><label for="swsales_hide_for_roles"><?php esc_html_e( 'Hide Sale by Role', 'sitewide-sales' ); ?></label></th>
```

Comment:
for attributes need to point to an id, not a name attribute. So probably change to swsales_hide_for_roles_select